fili-core/src/main/java/com/yahoo/bard/webservice/web/ErrorMessageFormat.java

```diff
@@ -216,6 +216,11 @@
     RESULT_MAPPING_FAILURE(
             "Error occurred while processing response data: %s"
     ),
+    INVALID_DATASOURCE_UNION(
+            "Union data source had conflicting name mappings for logical dimension '%s' with mappings of '%s' and '%s'",
+            "Union Data Source had conflicting name mappings for logical dimension '%s' with mappings of '%s' and '%s'"
```

Comment:
Do we need both? If the logging and exception message are exactly the same, why repeat it?